docs/FATE_Upgrade.md

```diff
@@ -0,0 +1,284 @@
+
+After installation, the pods are like:
+```
+client-75f67f5d47-svfh2           1/1     Running   0          2m30s
+clustermanager-7bd6fb46c8-bnx57   1/1     Running   0          2m30s
+mysql-77f95d4844-8dfjj            1/1     Running   0          2m30s
+nodemanager-0-6dbcc56dd4-ddldx    2/2     Running   0          2m30s
+nodemanager-1-646ddbf48c-2lllz    2/2     Running   0          2m30s
+python-5bd5d69779-c4t5r           2/2     Running   0          2m30s
+rollsite-5d45c85c4d-dbpnt         1/1     Running   0          2m30s
+```
+The PVC/PC are provisioned by the storage class, they are like:
+
+PVC:
+```
+NAMESPACE   NAME                 STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+fate-9999   client-data          Bound    pvc-e7ea7c79-c5e3-4824-8269-4eac390dcae8   1Gi        RWO            nfs-client     3m27s
+fate-9999   mysql-data           Bound    pvc-0865a743-9bfb-4990-bd26-84b0241f0084   1Gi        RWO            nfs-client     3m27s
+fate-9999   nodemanager-0-data   Bound    pvc-eaac241e-4d0f-4d06-96d0-30e85e284adc   2Gi        RWO            nfs-client     3m27s
+fate-9999   nodemanager-1-data   Bound    pvc-eed13a83-442e-4c9d-bb00-e6a7ce7c05f2   2Gi        RWO            nfs-client     3m27s
+fate-9999   python-data          Bound    pvc-5fc77a70-fe67-4b10-8db5-d3b0ef76d8f7   1Gi        RWO            nfs-client     3m27s
+```
+
+PV:
+```
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                          STORAGECLASS   REASON   AGE
+pvc-0865a743-9bfb-4990-bd26-84b0241f0084   1Gi        RWO            Retain           Bound    fate-9999/mysql-data           nfs-client              3m44s
+pvc-5fc77a70-fe67-4b10-8db5-d3b0ef76d8f7   1Gi        RWO            Retain           Bound    fate-9999/python-data          nfs-client              3m43s
+pvc-e7ea7c79-c5e3-4824-8269-4eac390dcae8   1Gi        RWO            Retain           Bound    fate-9999/client-data          nfs-client              3m44s
+pvc-eaac241e-4d0f-4d06-96d0-30e85e284adc   2Gi        RWO            Retain           Bound    fate-9999/nodemanager-0-data   nfs-client              3m44s
+pvc-eed13a83-442e-4c9d-bb00-e6a7ce7c05f2   2Gi        RWO            Retain           Bound    fate-9999/nodemanager-1-data   nfs-client              3m44s
+```
+
+### Backup the Mysql database
+Before any upgrade, we should back up the database. Currently, KubeFATE doesn't support back up the database automatically. This need to be done by the mysqldump tool manually. In the future this could be another feature of KubeFATE.
+
+There are several ways to do the backup work, in this example, we do that in the mysql pod:
+
+```
+$ k exec -it mysql-77f95d4844-8dfjj -n fate-9999 bash                       
```

Comment:
I assume "k" is the short hand you setting in your terminal. But it may cause confuse. And maybe we need to explain "mysql-77f95d4844-8dfjj" is my installing case, and how I know this.

Reply:
Fixed.